cluster/certificate/values.yaml

```diff
@@ -1,4 +1,3 @@
-defaultWildcardSecret: false
```

Comment:
And this!

Reply:
You can use `--set-string` to set properly the true/false flag.

Reply:
We don't need it any more